### ISSUE TITLE
ScreenName from Twitter or Name from Facebook Is no show

### ISSUE THREAD

**Issue by rizafahmi** [tsega/meteor-mentions#3]
This the code looks like

```
  $("textarea").mention({
    delimiter: '@',
    queryBy: ['name', 'username', 'services.facebook.name', 'services.twitter.screenName'],
    users: Users.find().fetch()
  });
```

But `services.facebook.name` or `services.twitter.screenName` is always no show?

Can you help me

**Comment by tsega** [comment]
@rizafahmi you are right, it won't work for nested document fields. The issue is that its expecting a [single level property](https://github.com/tsega/meteor-mentions/blob/master/lib/js/mention.js#L50),  I will see what I can do.

**Comment by tsega** [comment]
@rizafahmi OK now setting `queryBy` to a nested property, i.e. `services.twitter.screenName` should work just fine, but know this; it only works for property string with the dot `.` notation. Make sure that the property also exists! Let me know how it goes.

**Comment by rizafahmi** [comment]
@tsega thank you, but still not working on my end

**Comment by tsega** [comment]
Did you manage to update the package? If you do `meteor list` the version for the package should be `0.0.5`.

**Comment by rizafahmi** [comment]
Yes, i've check that in .meteor/version it's already 0.0.5

**Comment by rizafahmi** [comment]
@tsega any update on this? I tried to solve this, but doesn't quite understand the code, haha...

**Comment by tsega** [comment]
Can you open your browser console and see if there are any error there? If so kindly share them here.

**Comment by rizafahmi** [comment]
@tsega no error at all.

![make the city diy urban regeneration project of a city void by matching migrants and locals 2016-02-09 18-44-09](https://cloud.githubusercontent.com/assets/1147918/12915379/55754356-cf5d-11e5-810e-6c5de325e1c3.png)

**Comment by tsega** [comment]
In the array where you specify the list of fields can you place `screenName` at the beginning. E.g.

```
queryBy: ['services.twitter.screenName', 'name', 'username', 'services.facebook.name']
```

**Comment by rizafahmi** [comment]
Still the same @tsega :'(

**Comment by tsega** [comment]
Sorry about this, I'll get back to you later today.

**Comment by tsega** [comment]
Sorry about this, I'll get back to you later today.

**Comment by egemon** [comment]
Ahahah)) later today still here)
